### DISCUSSION TITLE
deconz e minipc

### DISCUSSION THREAD

**Discussion by H2opolo** [indomus/forum#2591] [HUB: Home Assistant]
Buongiorno
ho deciso di passare da raspberry 4 ad un mini pc (preso dalla vs pagina sulle occasioni)
Ho fatto prima una installazione con windows 11 e con vm ed è andato tutto bene
Poi ho provato ad installare sul mini pc la versione scaricata dal sito di "home assistant" per x86-64 generico (quella dove HA diventa sistema operativo) ed ho riscontrato un problema con deconz che non riesco a configurare. 
In particolare l' ad on si installa, parte (vede la chiavetta conbee II) ma se clicco sulla pagina web per andare su phoscon o non mi apre la pagina o non riesco a completare la procedura di registrazione 
Mi potete aiutare per piacere?
P

**Comment by KC-inDomus** [comment]
non si capisce bene quel che fai. innanzitutto, hai installato home assistant os: hai usato la nostra guida?
poi che intendi con andare alla pagina web su phoscon, cosa stai provando a installare e come?

**Reply by H2opolo** [reply]
Ho usato la guida presa dal sito di Home assistant per mini pc
dopo aver installato deconz bisogna registrare il gateway e collegarlo alla integrazione deconz
per fare questo dovrei accedere alla pagina web (jpg allegato) ma dopo aver cliccato sul tasto phoscon non riesco ad entrare
![deconz](https://github.com/indomus/forum/assets/75537956/d2e38a08-a103-43ef-a851-9efabd9b217a)

**Reply by KC-inDomus** [reply]
https://indomus.it/guide/come-installare-e-configurare-deconz-su-home-assistant-hassio/

a prescindere: usa zigbee2mqtt, non deconz
